### PR TITLE
Enforce empty line at the end of strings.yaml

### PR DESCRIPTION
**Pull request by flodolo**
No empty line means that the last string changes every time we add new content at the end of the file.

### CODE REVIEW

File: scripts/utils/generate_strings.py [hunk 1]
```diff
@@ -56,6 +56,11 @@ def parseTranslationStrings():
         exit("Unable to find translations/strings.yaml")
 
     with open(yaml_path, "r", encoding="utf-8") as yaml_file:
+        # Enforce a new line at the end of the file
```
Comment (flodolo) [inline scripts/utils/generate_strings.py:59]: This is the actual change.

File: scripts/utils/generate_strings.py [hunk 2]
```diff
@@ -133,7 +138,9 @@ def parseGuideStrings():
     for guide_filename in os.listdir(guides_path):
         if not guide_filename.endswith(".json"):
             continue
-        with open(os.path.join(guides_path, guide_filename), "r", encoding="utf-8") as guide_file:
+        with open(
```
Comment (flodolo) [inline scripts/utils/generate_strings.py:141]: This is just reformatting from black (in a separate commit)

**Review by bakulf — APPROVED** [review]

**Review by bakulf — CHANGES_REQUESTED** [review]

File: scripts/utils/generate_strings.py (hunk 1 above)
Comment (flodolo) [reply scripts/utils/generate_strings.py:59]: Bah, I even thought of checking that, and then forgot 🤦🏼 

What about using `.seek()`. Trying with that, but happy to create two read cycles instead.

Comment (bakulf) [reply scripts/utils/generate_strings.py:59]: `.seek()` works too.

**Review by bakulf — APPROVED** [review]

**Pull request merged**